assets/src/js/plugins/give-api/api.js

```diff
@@ -324,6 +324,15 @@ const Give = {
 			}
 			return url;
 		},
+
+		/**
+		 * Helper function used to determine if the given value is a float
+		 * @param value
+		 * @returns {boolean}
+		 */
+		isFloat: function( value ) {
+			return Number( value ) === value && value % 1 !== 0;
+		}
```

Comment:
I suggest coming about this slightly differently. In JS _all_ numbers are floats. That is, the only numeric primitive is Number, and Number is a 64-bit floating point value. So, really, all numbers in JS are floats. I believe what we're really checking for here is whether or not a Number has decimals:
```js
numberHasDecimals: function( value ) {
	return Math.floor( value ) === Number( value );
}
```

Reply:
@JasonTheAdams you are right, all numbers in JS are floats, so this function doesn't make much sense. I'll update this as you suggested.